rust/openvasd/src/controller/entry.rs

```diff
@@ -134,165 +107,128 @@ where
             }
         }
     }
+    tracing::debug!("{} {}", req.method(), kp);
```

Comment:
What is the second log for? These information are already contained in the log before